src/_includes/_layouts/BestPractices.jsx

```diff
@@ -25,7 +25,14 @@ export default function Home({ title, children, bestPractices, comp }) {
       </head>
       <body className="bg-gray-100 min-h-screen">
         <comp.Header />
-        <comp.GenericHero text="Best Practices" image_url="/assets/image.png" />
+        <comp.GenericHero
```

Comment:
I think the rebase or merge was done incorrectly, these changes were from my PR. It's fine to leave, but just wanted to mention for future reference.